### DISCUSSION TITLE
Wrong callbackUrl for blitz next-auth integration

### DISCUSSION THREAD

**Discussion by msShanky** [nextauthjs/next-auth#10119] [Help]
I have been breaking my head over this for couple of days. The requirement is to deploy this application built of from blitz project and when trying to change the port or the domain other than http://localhost:3000 nothing works

1. Deployed the site to vm with a custom domain - Did not work
2. Locally changed the port from 3000 -> 4000 with `yarn start -p 4000` - Did not work 

This is always pointing to localhost 3000 even after setting NEXTAUTH_URL, AUTH_URL and have also tried custom redirection with setting redirect_uri param while signin using NEXT_PUBLIC_APP_ORIGIN still does not work. 

![image](https://github.com/nextauthjs/next-auth/assets/19262066/7c111f33-c650-4f5a-a329-6fb8ae3b2aa8)

Has anybody faced something similar or any leads on this would be greatly helpful. Thanks

**Comment by Di0nza** [comment]
Did u solve it ? I have the same problem

**Reply by msShanky** [reply]
No, I had to drop blitz and move to using nextjs directly

**Comment by k-taro56** [comment]
next-auth or auth.js (next-authv5) ?

**Reply by msShanky** [reply]
I had trouble with v5 since it's in beta I went with next-auth v4
